frontend/src/metabase-lib/drills.ts

```diff
@@ -1,16 +1,24 @@
 import * as ML from "cljs/metabase.lib.js";
-import type { DataRow, Dimension, Query } from "./types";
+import { DatasetColumn } from "metabase-types/api";
+import type {
+  ColumnMetadata,
+  DataRow,
+  Dimension,
+  DrillThru,
+  Query,
+} from "./types";
 
+// Get a list (possibly empty) of available drill-thrus for a column, or a column + value pair
 // NOTE: value might be null or undefined, and they mean different things!
 // null means a value of SQL NULL; undefined means no value, ie. a column header was clicked.
 export function availableDrillThrus(
   // TODO: What is the right type for a JS column? (Not types.ts ColumnMetadata; that's the opaque CLJS type.)
   query: Query,
   stageIndex: number,
-  column: Record<string, unknown>,
+  column: ColumnMetadata | DatasetColumn | undefined,
   value: any,
```

Comment:
Should `value` have a [`RowValue` type](https://github.com/metabase/metabase/blob/bb00748b09aeb9fa219d08ea5bcff058ab223756/frontend/src/metabase-types/api/dataset.ts#L10)?